src/data/modals.ts

```diff
@@ -0,0 +1,24 @@
+export const generateKeyPairDataStep1 = {
```

Comment:
Avoid `data`

Reply:
Fixed in commit d5dd696371cd44e45af55bcbabd6811af380cde2.